Makefile

```diff
@@ -105,9 +105,11 @@ install: dev
 	@$(VENV) luarocks make
 
 clean: check-bazel
```

Comment:
Should we remove `check-bazel`? As we are cleaning stuffs, no need to install `bazelisk`.

Reply:
still need bazel to do clean, i will move `rm -f bin/bazel bin/grpcurl` to after `$(BAZEL) clean`